### DISCUSSION TITLE
Does the windows build require MacOS specific hints?

### DISCUSSION THREAD

**Discussion by AntonKrug** [eclipse-embed-cdt/eclipse-plugins#604] [General Discussions]
Is there a need to have MacOS advice when running Windows build of the eclipse?
![image](https://github.com/eclipse-embed-cdt/eclipse-plugins/assets/30338596/5a741e48-019e-4d1e-a354-a5b405a7de46)

**Comment by ilg-ul** [comment]
Well, the message is probably not very fortunate.

The advice is basically to configure the file manager in such a way that hidden folders are visible. On macOS the trick is to use Shift+Cmd+','; I have no idea how to do it on Windows; neither on Linux... :-(

**Comment by TommyMurphyTM1234** [comment]
FWIW

> I have no idea how to do it on Windows

Windows Explorer: https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5

> neither on Linux...

I think that `Ctrl-H` works on many distros these days but it may depend on the specific desktop environment/filesystem explorer.

**Reply by ilg-ul** [reply]
Thank you, Tommy.

If you have a better phrasing for that message, PRs are welcome.
